dask_jobqueue/core.py

```diff
@@ -484,8 +484,15 @@ def __init__(
             "dashboard_address": ":8787",
             "security": security,
         }
-        # scheduler_options overrides parameters common to both workers and scheduler
-        scheduler_options = dict(default_scheduler_options, **scheduler_options)
+        
+        config_scheduler_options = dask.config.get("jobqueue.%s" % config_name).get('scheduler_options') or {}
```

Comment:
I am not 100% sure but I would do something like this: 
```py
if scheduler_options is None:
    scheduler_options = dask.config.get("jobqueue.%s" % config_name, {})
    
scheduler_options = dict(default_scheduler_options, **scheduler_options)
```

What I am aiming at here is that the `scheduler_options` override completely the config `scheduler_options` rather than trying to combine both and have some keys set from the config and some key set from the parameter.

To me it feels simpler and less error-prone this way.

Reply:
> What I am aiming at here is that the scheduler_options override completely the config scheduler_options rather than trying to combine both and have some keys set from the config and some key set from the parameter.

Thanks for putting that up! Actually, I had implemented what I did there very intentionally, because I like to have things flexible. However, I just had a look on how Dask jobqueue handles this for other, i.e. the (variable length) list-type parameters. No merge happening there, so I have decided to implement your suggestion to keep this consistent.